### PR TITLE
feat: Lemonify project settings

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

Project Settings heavily uses Antd components and has some awkward bugs. Rather than completely rework it in the bigger way planned, I focused on simply removing Antd in favour of our standard components.

## Changes

* Only things **not** Lemonified are the PropertyFilters (this would be a more invasive change as it is used all over Insights) and some icons

**Side effect - improves Path filtering view**
|Before|After|
|----|----|
|Screenshot 2022-08-11 at 12 40 08|Screenshot 2022-08-11 at 12 39 50|

**Long screenshot of Project Settings**

Screenshot 2022-08-11 at 12 48 27

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

Entirely in the UI

### CODE REVIEW

File: frontend/src/lib/components/LemonModal/LemonModal.scss [hunk 1]
```diff
@@ -77,7 +77,7 @@
 
     h3 {
         margin-bottom: 0;
-        margin-right: 1rem;
+        margin-right: 1.5rem;
```
Comment (benjackwhite) [inline frontend/src/lib/components/LemonModal/LemonModal.scss:80]: This accounts better for the close icon in the modal

File: frontend/src/lib/components/PersonPropertySelect/PersonPropertySelect.tsx [hunk 2]
```diff
@@ -23,30 +25,22 @@ const PropertyTag = ({
...
-                padding: '0.25rem 0.5em',
-                background: '#D9D9D9',
-                border: '1px solid #D9D9D9',
-                borderRadius: '40px',
-                fontSize: 'inherit',
-                cursor: sortable ? 'move' : 'auto',
+    <span className={clsx(sortable ? 'cursor-move' : 'cursor-auto')}>
+        <LemonSnack
+            onClose={() => {
+                console.log('closed')
```
Comment (pauldambra) [inline frontend/src/lib/components/PersonPropertySelect/PersonPropertySelect.tsx:31]: 😱

File: frontend/src/scenes/project/Settings/TimezoneConfig.tsx [hunk 3]
```diff
@@ -15,42 +16,41 @@ export function TimezoneConfig(): JSX.Element {
         return <Skeleton paragraph={{ rows: 0 }} active />
     }
     function onChange(val: string): void {
-        Modal.confirm({
+        console.log(val)
```
Comment (pauldambra) [inline frontend/src/scenes/project/Settings/TimezoneConfig.tsx:19]: 😱

File: frontend/src/scenes/project/Settings/TimezoneConfig.tsx [hunk 4]
```diff
@@ -15,42 +16,41 @@ export function TimezoneConfig(): JSX.Element {
...
             title: `Do you want to change the timezone of this project?`,
-            content:
+            description:
                 'This will change how every graph in this project is calculated, which means your data will look different than it did before.',
-            icon: <ExclamationCircleOutlined />,
-            okText: 'Change timezone',
-            okType: 'danger',
-            onOk() {
-                updateCurrentTeam({ timezone: val })
+            // icon: <ExclamationCircleOutlined />,
```
Comment (pauldambra) [inline frontend/src/scenes/project/Settings/TimezoneConfig.tsx:24]: ??

**Review by pauldambra — APPROVED** [review]
few things to tidy but otherwise 🦈

**Review by clarkus — APPROVED** [review]

**Pull request merged**